service/src/test/java/de/adorsys/psd2/sandbox/xs2a/model/Context.java

```diff
@@ -13,6 +13,7 @@
   private String paymentProduct;
   private String paymentService;
   private String paymentId;
+  private String accountId;
```

Comment:
List of accountIds?

Reply:
Is not needed, because cucumber tests the lifecycle of a expected request. It does not bring a lot of benefits to request transactions of another account of a psu since we already know that the other account has exact the same structure than the first account. So the behaviour is also similar.